python/GangaLHCb/testlib/setup_env.sh

```diff
@@ -1,2 +1,6 @@
 source /cvmfs/lhcb.cern.ch/lib/LbLogin.sh
-SetupProject ganga
+export GANGA_CONFIG_PATH=GangaLHCb/LHCb.ini
+export GANGA_SITE_CONFIG_AREA=/cvmfs/lhcb.cern.ch/lib/GangaConfig/config
+ls -l .globus
+lhcb-proxy-init
+#SetupProject ganga
```

Comment:
Before there was a ```SeupProject``` command here that effectively was used to get the dependencies included and setting the environment variables. The ```lhcb-proxy-init`` is because I have no idea about how the proxy was created before ...